fuse/cross_agency.py

```diff
@@ -172,9 +179,14 @@ def cross_match_officers_between_agencies(personnel, events, constraints):
     return clusters, per[["max_timestamp", "agency"]]
 
 
-def create_person_table(clusters, personnel, personnel_event):
+def create_person_table(clusters, personnel, personnel_event, post):
     # add back unmatched officers into clusters list
-    matched_uids = frozenset().union(*[s for s in clusters])
+    matched_uids_post = frozenset().union(*[s for s in post["uids"]])
+    matched_uids_clusters = frozenset().union(*[s for s in clusters])
+
+    list_matched_uids = [matched_uids_post, matched_uids_clusters]
+    matched_uids = frozenset().union(*list_matched_uids)
```

Comment:
Not sure how well this works but I prefer assigning a `history_id` that is unique per history record. And then add `AbsoluteScorer("history_id", 1, ignore_key_error=True),` to the `MaxScorer` which essentially make sure all uids that share the same history_id always match.